src/ansys/geometry/core/tools/problem_areas.py

```diff
@@ -131,11 +178,19 @@ def fix(self) -> RepairToolMessage:
         response = self._repair_stub.FixDuplicateFaces(
             FixDuplicateFacesRequest(duplicate_face_problem_area_id=self._grpc_id)
         )
```

Comment:
Oh I just realized about this.. we need to move this into the gRPC folder implementation instead of having at this level raw gRPC requests. That would avoid method duplication as well for serialization.

Let's do this in a follow up PR I think